packages/pieces/date-helper/src/lib/actions/next-day-of-week.ts

```diff
@@ -0,0 +1,97 @@
+import { Property, createAction, Validators } from '@activepieces/pieces-framework';
+import { optionalTimeFormats, timeFormat, timeFormatDescription, createNewDate, timeZoneOptions, timeDiff } from '../common';
+
+export const nextDayofWeek = createAction({
+	name        : 'nextDayofWeek',
```

Comment:
Action and trigger names should be snake_case

Reply:
All done.